spec/support/login_helper.rb

```diff
@@ -0,0 +1,13 @@
+module LoginHelper
+  def login(user)
+    visit new_user_session_path
+
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+
+    click_button 'Log in'
+
+    expect(page).to have_content('Signed in successfully')
```

Comment:
Generally speaking, `helper` methods should contain expectations. 

`login` is being used in a lot of places,  and each time it gets called the tests would run two additional expectations. This could conceivably slow down your test suite, and weirdly increase your test numbers.

These specific expectations have coverage in the `user_logs_in` spec, so you're still testing basic login behavior.